### PR TITLE
Docs: Document helidon-health-checks maven dependency in MP Health docs

### PR DESCRIPTION
**Pull request by barchetta**

### CODE REVIEW

File: docs/mp/health/01_introduction.adoc [hunk 1]
```diff
@@ -36,6 +36,17 @@ include::{common-deps-page-prefix-inc}[tag=maven-dependency]
         </dependency>
 ----
 
+To enable built-in health checks add the following dependency
+(or us the <<mp/introduction/02_microprofile.adoc, helidon-microprofile bundle>> )
```
Comment (tjquinno) [inline docs/mp/health/01_introduction.adoc:40]: typo: missing 'e' at the end of "use"

File: docs/mp/health/01_introduction.adoc [hunk 2]
```diff
@@ -36,6 +36,17 @@ include::{common-deps-page-prefix-inc}[tag=maven-dependency]
...
+
+[source,xml]
+----
+        <dependency>
+            <groupId>io.helidon.microprofile.health</groupId>
```
Comment (tjquinno) [inline docs/mp/health/01_introduction.adoc:45]: The `groupId` is `io.helidon.health` (no `microprofile`).

**Review by tjquinno — CHANGES_REQUESTED** [review]
Two corrections

File: docs/mp/health/01_introduction.adoc (hunk 1 above)
Comment (barchetta) [reply docs/mp/health/01_introduction.adoc]: Fixed

File: docs/mp/health/01_introduction.adoc (hunk 2 above)
Comment (barchetta) [reply docs/mp/health/01_introduction.adoc]: Fixed

**Review by tjquinno — APPROVED** [review]

**Pull request merged**